app/src/main/java/grinnell/appdev/edu/lyles/MenuViewPagerAdapter.java

```diff
@@ -49,7 +43,7 @@ public CharSequence getPageTitle(int position) {
             case SNACKS_INDEX: return mResources.getString(R.string.snacks_tab_title);
             case DRINKS_INDEX: return mResources.getString(R.string.drinks_tab_title);
             case FAVORITES_INDEX: return mResources.getString(R.string.favorites_tab_title);
-            default: return new String();
+            default: return "";
```

Comment:
So assuming you have a file for constants already, we can just create a new `String` there like: `private static final String DEFAULT_ERR_MSG = "Non-existent Fragment at given position"` and then just call: `Constants.DEFAULT_ERR_MSG` (use static imports to shorten the code!) in the `default` case